lldb/source/Plugins/ExpressionParser/Clang/ClangUserExpression.h

```diff
@@ -185,9 +185,9 @@ class ClangUserExpression : public LLVMUserExpression {
   /// The parameter have the same meaning as in ClangUserExpression::Parse.
   /// \see ClangUserExpression::Parse
   bool TryParse(DiagnosticManager &diagnostic_manager,
-                ExecutionContextScope *exe_scope, ExecutionContext &exe_ctx,
-                lldb_private::ExecutionPolicy execution_policy, bool keep_result_in_memory,
-                bool generate_debug_info);
+                ExecutionContext &exe_ctx,
+                lldb_private::ExecutionPolicy execution_policy,
+                bool keep_result_in_memory, bool generate_debug_info);
```

Comment:
Can we revert this and always just pass in "exe_ctx.GetBestExecutionContextScope()"?